plugins/drill-coverage-plugin/src/adminPartTest/kotlin/MethodsComparator.kt

```diff
@@ -0,0 +1,54 @@
+package com.epam.drill.plugins.coverage.test
+
+import com.epam.drill.plugins.coverage.*
+import java.io.*
+import java.nio.file.*
+import kotlin.test.*
+
+class MethodsComparatorTest {
+
+    private val stubsDir = "${Paths.get(".").toAbsolutePath().normalize()}\\build\\processedResources\\adminPart\\test\\"
```

Comment:
Need some help with this. Drone can't find my files when running these tests. Failed to figure out how to give this monster a correct path